packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -765,6 +765,10 @@ export default class TradeStore extends BaseStore {
                 }
             }
 
+            if (obj_new_values.contract_type && obj_new_values.contract_type === 'rise_fall' && !!this.is_equal) {
+                this.contract_type = 'rise_fall_equal';
```

Comment:
```suggestion
                obj_new_values.contract_type = 'rise_fall_equal';
```
We do further processing in `processTradeParams` with the `obj_new_values`, so we should set it in `obj_new_values` and I think it's better not to change 
`contract_type` in trade store directly.

Reply:
You are right, this needs to be updated to `obj_new_values `.